kedro-org/kedro-viz
# Feature/improve metadata panel node naming

## Description

Resolves the metadata-panel part of #1252. 

@idanov had the great idea of always showing all node names (pretty name/original name) in the metadata panel. This does that.

## QA notes

Look at the Gitpod link and see the update in the panel. 

We may need to decide on what the text should be for the labels. I've gone with `Pretty node name` and `Original node name`. 

<img width="388" alt="image" src="https://user-images.githubusercontent.com/16869061/216962150-8c3c1bb7-35d7-4bd3-8ca6-2bf63f8a559e.png">

@AntonyMilneQB you care a lot about language. Do you have any thoughts here?

## Checklist
